core/rio/ntriples/src/main/java/org/eclipse/rdf4j/rio/ntriples/NTriplesUtil.java

```diff
@@ -7,19 +7,14 @@
  *******************************************************************************/
 package org.eclipse.rdf4j.rio.ntriples;
 
-import java.io.IOException;
-
-import org.eclipse.rdf4j.model.BNode;
-import org.eclipse.rdf4j.model.IRI;
-import org.eclipse.rdf4j.model.Literal;
-import org.eclipse.rdf4j.model.Resource;
-import org.eclipse.rdf4j.model.Value;
-import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.*;
```

Comment:
Minor thing: we don't use wildcard imports. Can you tweak your editor settings to always expand imports?